tests/ui/type-alias-impl-trait/higher_kinded_params2.rs

```diff
@@ -16,12 +16,14 @@ impl<'a> B for &'a A {
 }
 
 type Tait = impl std::fmt::Debug;
+//~^ ERROR: unconstrained opaque type
 
 struct Terminator;
 
 type Successors<'a> = impl std::fmt::Debug + 'a;
 
 impl Terminator {
+    #[define_opaques(Successors)]
```

Comment:
```suggestion
    #[define_opaques(Successors, Tait)]
```
or yeet this test. Rn the description does not match the actual behavior :thinking:

Reply:
Yea it hasn't been doing that for a while. I think I broke it with the signature rule